main_robot:
  beacon_detector_node:
    ros__parameters:
      enable_robot_position_reception: False
      object_topic: object
      position_topic: position_out
      debug_topic: debug
      color_topic: init/team_color
      default_color: yellow
      beacons: [0.072, 3.094, 1.928, 3.094, 1., -0.094, -0.12, 1.725]
      boundaries:  [0.0, 2.0, 0.0, 3.0]
      init_position: [1.77, 0.23, 0.0]
      distance_tolerance: 0.3
      angle_tolerance: 0.7853981633974483
      display_topic: top_lidar_position_display
      robot_position_topic: robot_position_input
      enable_wait_color: False
      enable_initial_position: False

  beacon_generator_node:
    ros__parameters:
      object_topic: object
      moveto_action: moveto
      real_position_topic: real_position
      color_service: init/team_color
      team_color: yellow
      random_moves: False
      beacons: [0.072, 3.094, 1.928, 3.094, 1., -0.094, -0.12, 1.725]
      boundaries:  [0.0, 2.0, 0.0, 3.0]
      period: 0.5
      angular_velocity: 0.5
      linear_velocity: 0.1